### PR TITLE
[docker-syncd-brcm-rpc]: Fix build

### PR DESCRIPTION
**Pull request by pavel-shirshov**

### CODE REVIEW

File: dockers/docker-syncd-brcm-rpc/Dockerfile [hunk 1]
```diff
@@ -43,7 +43,7 @@ RUN apt-get update \
 COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 COPY ptf_nn_agent.conf /etc/supervisor/conf.d/ptf_nn_agent.conf
 
-ENTRYPOINT service rsyslog start \
-    && sysctl -w net.core.rmem_max=509430500 \
-    && service syncd start       \
-    && /usr/bin/supervisord
+RUN sed -i 's/read/sysctl -w net.core.rmem_max=509430500 ; \/usr\/bin\/supervisord/' /usr/bin/start.sh
```
Comment (jleveque) [inline dockers/docker-syncd-brcm-rpc/Dockerfile]: I like what you're doing here, but all of the other start.sh files are checked in to the project. Just wondering if you should follow suit here for uniformity's sake. If the file is present in the repo, it's obvious it should be used.

Comment (taoyl-ms) [inline dockers/docker-syncd-brcm-rpc/Dockerfile]: Why do we have this change only for brcm but not for mlnx?

Comment (pavel-shirshov) [inline dockers/docker-syncd-brcm-rpc/Dockerfile]: Added

Comment (jleveque) [inline dockers/docker-syncd-brcm-rpc/Dockerfile]: Probably should change the title of the PR to reflect this.

**Comment by lguohan** [comment]
can you build these two dockers in image build process so that we can build the docker in the buildimage process? need to add a ```docker-syncd-brcm-rpc.mk``` in platform/broadcom directory and add it to ```rules.mk```. The same for mlnx version.

**Comment by pavel-shirshov** [comment]
ok

**Comment by pavel-shirshov** [comment]
@lguohan https://github.com/Azure/sonic-buildimage/pull/431

**Pull request merged**